lib/CMakeLists.txt

```diff
@@ -141,6 +141,7 @@ set(LLVM_INCLUDE_BENCHMARKS OFF CACHE BOOL "ponyc specific override of LLVM cach
 set(LLVM_INCLUDE_TESTS OFF CACHE BOOL "ponyc specific override of LLVM cache entry")
 set(LLVM_TOOL_REMARKS_SHLIB_BUILD OFF CACHE BOOL "ponyc specific override of LLVM cache entry")
 set(LLVM_ENABLE_ZLIB OFF CACHE BOOL "ponyc specific override of LLVM cache entry")
+set(LLVM_ENABLE_LIBXML2 "OFF" CACHE STRING "ponyc specific override of LLVM cache entry")
```

Comment:
why is this string when the others are BOOL?

Reply:
the llvm cmake option was typed as STRING, but it also works as BOOL. changed it for the sake of consistency.